publications-2020.bib

```diff
@@ -0,0 +1,15 @@
+% Encoding: US-ASCII
+
+@article{10.1093/gji/ggaa039,
+    author = {Käufl, J S and Grayver, A V and Comeau, M J and Kuvshinov, A V and Becken, M and Kamm, J and Batmagnai, E and Demberel, S},
```

Comment:
It's unhappy with the umlaut. Can you write it as `\"a`?

Reply:
Fixed